frontend/packages/ux-editor/src/utils/stringUtils/stringUtils.ts

```diff
@@ -0,0 +1,7 @@
+export const stringToArray = (value: string): string[] => {
+  return value?.replace(/ /g, '').split(',');
```

Comment:
Det er kanskje ikke aktuelt i tilfellet hvor denne brukes nå, men generelt er det muligens mer hensiktsmessig å kun fjerne mellomrommene på starten og slutten av hvert element?
```suggestion
  return value?.split(',').map((s) => s.trim());
```